recipes/ipyurl/meta.yaml

```
@@ -0,0 +1,54 @@
+{% set version = "0.1.2" %}
+{% set sha256 = "9831f53ed6c564b59d72a047e627785139069e4bc0311d3393fd9fc6f52389bb" %}
+
+package:
+  name: ipyurl
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/i/ipyurl/ipyurl-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
```

Please omit the Jinja here.
```suggestion
  sha256: 9831f53ed6c564b59d72a047e627785139069e4bc0311d3393fd9fc6f52389bb
```